package.json

```diff
@@ -24,27 +31,27 @@
       "limit": "2 KB"
     },
     {
-      "path": "rgb/index.module.js",
+      "path": "rgb/dist/index.module.js",
       "limit": "2 KB"
     },
     {
-      "path": "rgbString/index.module.js",
+      "path": "rgbString/dist/index.module.js",
       "limit": "2 KB"
     },
     {
-      "path": "hsl/index.module.js",
+      "path": "hsl/dist/index.module.js",
       "limit": "2 KB"
     },
     {
-      "path": "hslString/index.module.js",
+      "path": "hslString/dist/index.module.js",
       "limit": "2 KB"
     },
     {
-      "path": "hsv/index.js",
+      "path": "hsv/dist/index.module.js",
```

Comment:
I changed this from `index.js` to `index.module.js` as I assumed it was a mistake? Every other path was for `.../index.module.js`. Let me know if it wasn't and I can revert.

Reply:
Good catch!